addons/mail/views/mail_link_preview_views.xml

```diff
@@ -16,6 +16,7 @@
                         </group>
                         <group>
                             <field name="og_image"/>
+                            <field name="og_image" nolabel="1" widget="image_url" options="{'size': [400, 400]}" />
```

Comment:
400 is too much IMO

![image](https://github.com/user-attachments/assets/3d304412-c1ca-4ace-89f6-f8cbdbe92d14)